fe/fe-core/src/main/java/com/starrocks/common/FeNameFormat.java

```diff
@@ -43,11 +43,11 @@ public class FeNameFormat {
     private FeNameFormat() {}
 
     private static final String LABEL_REGEX = "^[-\\w]{1,128}$";
-    public static final String COMMON_NAME_REGEX = "^[a-zA-Z]\\w{0,63}$|^_[a-zA-Z0-9]\\w{0,62}$";
-    // The mysql protocol does not limit the length of the db, but the buffer of the mysql-client is 512+1,
-    // minus the use and spaces, a total of 4 bytes, and finally can only limit the length of 509,
-    // in order to be compatible with the mysql-client, the limit length is 509.
-    public static final String DB_NAME_REGEX = "^[a-zA-Z]\\w{0,508}$|^_[a-zA-Z0-9]\\w{0,507}$";
+    public static final String COMMON_NAME_REGEX = "^[a-zA-Z]\\w{0,1024}$|^_[a-zA-Z0-9]\\w{0,1023}$";
```

Comment:
It is best not to let go of too many restrictions at once, which may cause some unexpected problems.